AmethystAPI/src/minecraft/src/common/world/level/block/Block.hpp

```diff
@@ -35,6 +35,9 @@ class Block : public BlockComponentStorage {
     template <typename T>
     gsl::strict_not_null<const Block*> setState(const BlockState& stateType, T value) const;
 
+    // Just a partial reimplementation using `BlockLegacy::getMapColor`
+    // Needed because `BlockLegacy::getMapColor` is a protected function and the Block class
```

Comment:
Simple is better, just do something like my other comment

Reply:
this should be covered now?
I mean I simplfied it ig...